Composer/packages/server/src/middleware/logNetworkTraffic.ts

```diff
@@ -18,9 +18,11 @@ export function logNetworkTraffic(req: Request, res: Response, next?: NextFuncti
   // when the request finishes, log the payload and status code to the client
   res.once('finish', () => {
     let data: ConversationNetworkErrorItem | ConversationNetworkTrafficItem | undefined;
+    let payload = (res as any).sentData;
+    payload = typeof payload === 'string' ? JSON.parse(payload || '{}') : payload;
```

Comment:
Wrapping anything with JSON.parse around a try catch would be defensive. JSON.parse(null/undefined) would throw